dask_cuda/benchmarks/local_cudf_merge.py

```diff
@@ -25,12 +24,20 @@
 # <https://gist.github.com/rjzamora/0ffc35c19b5180ab04bbf7c793c45955>
 
 
-def generate_chunk(i_chunk, local_size, num_chunks, chunk_type, frac_match, gpu):
+# Set default shuffle method to "tasks"
+if dask.config.get("dataframe.shuffle.method", None) is None:
+    dask.config.set({"dataframe.shuffle.method": "tasks"})
+
+
+def generate_chunk(input):
+    i_chunk, local_size, num_chunks, chunk_type, frac_match, gpu = input
+
     # Setting a seed that triggers max amount of comm in the two-GPU case.
     if gpu:
         import cupy as xp
 
         import cudf as xdf
+        import dask_cudf  # noqa: F401
```

Comment:
Generating a collection in this way puts us in a strange edge case where `dask_cudf` may never get imported on the worker, and so some type-based dispatching may fail. The only way to avoid this possibility is to import `dask_cudf` before we deploy the cluster, or to import `dask_cudf` here.